### ISSUE TITLE
Looking for maintainers?

### ISSUE THREAD

**Issue by chrisnicola** [foretagsplatsen/numbro#226]
I've noticed a lot of open pull requests. Is there any interest in getting some volunteer maintainers to help out? I'd be happy to contribute some time.

**Comment by BenjaminVanRyseghem** [comment]
That would be quite nice indeed :smile:

I usually try to answer every PR, but I am not sure there are many open PR that are actually mergeable, or ?

Anyway, help is always welcome

**Comment by chrisnicola** [comment]
@BenjaminVanRyseghem some look like they are, I would suggest having a policy for PRs such that after X days of no response and they are not mergeable or unclear then close them.

I do have a couple open recently that I'd love feedback on and I'm happy to update them quickly if they need any changes.

**Comment by BenjaminVanRyseghem** [comment]
I usually try to bump them.

Closing feel a bit harsh, or?

**Comment by mguida22** [comment]
I'd be happy to help contribute as well

**Comment by BenjaminVanRyseghem** [comment]
thanks :smile:

**Comment by chrisnicola** [comment]
@BenjaminVanRyseghem I don't think it is harsh. It keeps things clean. Someone can always update the code and ask that it be re-opened if they come back to it.

**Comment by BenjaminVanRyseghem** [comment]
I promise I do my best :smile:
